jekyll/_includes/sidebar.html

```diff
@@ -2,8 +2,10 @@
 <div class="sidebar-wrapper">
 	<nav class="sidebar fixed">
 		{% assign sidenav = site.data.sidenav %}
+		{% assign current_url = page.url | slice: 1, page.url.size %}
 
 		{% for main-item in sidenav %}
+		  {% assign current_url = page.url | slice: 1, page.url.size %}
```

Comment:
Does this need to be defined in here? It seems like the def on line 5 should be sufficient